Add VFS properties content (#1072)
wso2/docs-mi: en/docs/reference/synapse-properties/transport-parameters/vfs-transport-parameters.md

            When this parameter is set to <code>true</code>, the newly created file will have the same last-modified timestamp as the original file. The default setting is <code>true</code>.
         </td>
      </tr>
      <tr>
         <td>transport.vfs.CheckSizeIgnoreEmpty</td>
         <td>

>             When this parameter is set to <code>true</code>, empty files are ignored when reading. The default setting is <code>false</code>.

Reviewer: ```suggestion
            When this parameter is set to <code>true</code>, empty files are ignored during reading. The default setting is <code>false</code>.
```

         </td>
      </tr>
      <tr>
         <td>transport.vfs.CheckSizeInterval</td>
         <td>

>             The time duration in milliseconds between the size check on files.

Reviewer: ```suggestion
            The duration in milliseconds between the size checks on files.
```

         </td>
      </tr>
      <tr>
         <td>transport.vfs.MinimumAge</td>
         <td>

>             The minimum time duration in seconds for file age since last modified time.

Reviewer: ```suggestion
            The minimum duration in seconds for file age since the last modified time.
```

         </td>
      </tr>
      <tr>
         <td>transport.vfs.MaximumAge</td>
         <td>

>             The maximum time duration in seconds for file age since last modified time.

Reviewer: ```suggestion
            The maximum duration in seconds for file age since the last modified time.
```
